.github/workflows/wheels.yaml

```diff
@@ -25,32 +26,24 @@ jobs:
       - uses: actions/setup-python@v2
         name: Install Python
         with:
-          python-version: 3.9
-
-      - uses: s-weigand/setup-conda@v1
-        if: matrix.os == 'windows-latest'
-        with:
-          update-conda: true
-          conda-channels: conda-forge
-          activate-conda: true
-          python-version: 3.9
+          python-version: '3.10'
```

Comment:
Why do you change the environment that's used to run `cibuildwheel`? That's completely unnecessary.

Reply:
thanks for clarifying, I wasn't sure what was needed here since it's _currently_ using the latest supported Python version